crates/egui-winit/src/lib.rs

```diff
@@ -172,6 +172,16 @@ impl State {
         self.egui_input.max_texture_side = Some(max_texture_side);
     }
 
+    /// Returns the used clipboard.
+    pub fn clipboard(&mut self) -> &mut clipboard::Clipboard {
+        &mut self.clipboard
+    }
+
+    /// Returns whether the window should get IME events.
+    pub fn allow_ime(&self) -> bool {
```

Comment:
In my variant of the PR, this was mut because we need to _update_ this field :wink: